python/cloudtik/runtime/ml/utils.py

```diff
@@ -57,7 +57,7 @@ def publish_service_uri(cluster_config: Dict[str, Any], head_node_id: str) -> No
 
     provider = _get_node_provider(cluster_config["provider"], cluster_config["cluster_name"])
     head_internal_ip = provider.internal_ip(head_node_id)
-    service_uris = {"mlflow-service-uri": "http://{}:5001".format(head_internal_ip)}
+    service_uris = {"mlflow-service-uri": "{}:5001".format(head_internal_ip)}
```

Comment:
I guess you removed http:// because aliyun tag value doesn't allow this.
Instead of change the value, can we do the similar encoding and decoding like GCP so that any value can be set.

Reply:
@jerrychenhf Good suggestion! We can add the similar encoding and decoding logic for publish_global_variables and subscribe_global_variables function of AliyunWorkspaceProvider.